2021-12-13 20:27:09,458 INFO    MainThread:5792 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 20:27:09,458 INFO    MainThread:5792 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 20:27:09,458 INFO    MainThread:5792 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_202709-29wwyec3\logs\debug.log
2021-12-13 20:27:09,458 INFO    MainThread:5792 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_202709-29wwyec3\logs\debug-internal.log
2021-12-13 20:27:09,459 INFO    MainThread:5792 [wandb_init.py:init():404] calling init triggers
2021-12-13 20:27:09,459 INFO    MainThread:5792 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 20:27:09,461 INFO    MainThread:5792 [wandb_init.py:init():449] starting backend
2021-12-13 20:27:09,461 INFO    MainThread:5792 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 20:27:09,466 INFO    MainThread:5792 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 20:27:09,553 INFO    MainThread:5792 [backend.py:ensure_launched():204] started backend process with pid: 9416
2021-12-13 20:27:09,553 INFO    MainThread:5792 [wandb_init.py:init():458] backend started and connected
2021-12-13 20:27:09,555 INFO    MainThread:5792 [wandb_init.py:init():519] updated telemetry
2021-12-13 20:27:09,577 INFO    MainThread:5792 [wandb_init.py:init():549] communicating current version
2021-12-13 20:27:10,139 INFO    MainThread:5792 [wandb_init.py:init():554] got version response 
2021-12-13 20:27:10,139 INFO    MainThread:5792 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 20:27:10,360 INFO    MainThread:5792 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 20:27:10,470 INFO    MainThread:5792 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 20:27:10,472 INFO    MainThread:5792 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 20:27:10,472 INFO    MainThread:5792 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 20:27:10,474 INFO    MainThread:5792 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 20:27:10,474 INFO    MainThread:5792 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 20:27:10,475 INFO    MainThread:5792 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002A780088970>>
2021-12-13 20:27:10,475 INFO    MainThread:5792 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2021-12-13 20:27:10,476 INFO    MainThread:5792 [wandb_config.py:__setitem__():144] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002A780088970>>
2021-12-13 20:27:10,476 INFO    MainThread:5792 [wandb_run.py:_config_callback():962] config_cb batch_size 32 None
2021-12-13 20:27:14,115 INFO    MainThread:5792 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 20:27:14,115 INFO    MainThread:5792 [wandb_run.py:_restore():1758] restore
2021-12-13 20:27:14,596 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 948
  total_bytes: 948
}

2021-12-13 20:27:14,841 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 948
  total_bytes: 948
}

2021-12-13 20:27:15,399 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 948
  total_bytes: 3208
}

2021-12-13 20:27:15,501 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 948
  total_bytes: 3234
}

2021-12-13 20:27:15,612 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 948
  total_bytes: 3234
}

2021-12-13 20:27:15,719 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3234
  total_bytes: 3234
}

2021-12-13 20:27:15,827 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3234
  total_bytes: 3234
}

2021-12-13 20:27:15,937 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3234
  total_bytes: 3234
}

2021-12-13 20:27:16,048 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3234
  total_bytes: 3234
}

2021-12-13 20:27:16,160 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3234
  total_bytes: 3234
}

2021-12-13 20:27:16,684 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3234
  total_bytes: 3234
}

2021-12-13 20:27:17,197 INFO    MainThread:5792 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3234
  total_bytes: 3234
}
local_info {
}

2021-12-13 20:27:18,272 INFO    MainThread:5792 [wandb_run.py:_append_files():2184] logging synced files
